client/src/pages/Dashboard/Dashboard.tsx

```diff
@@ -24,11 +27,32 @@ export default function Dashboard(): JSX.Element {
   return (
     <PageContainer>
       <Grid container>
-        <Grid xs={12} item>
-          <Typography sx={{ textAlign: 'center' }} variant="h4">
-            Search Profiles
+        <Grid item xs={12} justifyContent="space-around">
+          <Typography sx={{ textAlign: 'center', paddingBottom: '16px' }} variant="h4">
+            All Profiles
           </Typography>
         </Grid>
+        {profiles.map((profile) => (
+          <Grid
+            item
+            key={profile.userId}
+            xs={4}
+            sx={{ display: 'flex', justifyContent: 'center', alignItems: 'center' }}
+          >
+            <ProfileCard profile={profile} />
+          </Grid>
+        ))}
+        <Grid item xs={12} textAlign="center">
+          <Button
+            size="large"
+            variant="outlined"
+            onClick={() => {
+              console.log('show more');
+            }}
```

Comment:
We can just leave the click handler off for now, we'll do this for pagination when the time comes.

Reply:
Sure Ethan, I'll take it off. I'm curious, is there some kind of standard to mark things for later? Maybe I should put in a ```// TODO``` instead? This goes to your other comment as well, considering that console log was useful for integration

Reply:
I think just as a team we'll know it's for integration 😄